test/llvmpasses/throw-amdgpu.jl

```diff
@@ -0,0 +1,52 @@
+# This file is a part of Julia. License is MIT: https://julialang.org/license
+
+# RUN: julia --startup-file=no %s | opt -load libjulia%shlibext -mtriple=amdgcn-amd-amdhsa -mcpu=gfx900 -AllocOpt -LateLowerGCFrame -S - | FileCheck %s
+
+isz = sizeof(UInt) == 8 ? "i64" : "i32"
+
+println("""
+%jl_value_t = type opaque
```

Comment:
If you are not doing an end-to-end test you don't need to use `.jl`, better just directly test the llvmir like https://github.com/JuliaLang/julia/blob/master/test/llvmpasses/muladd.ll

Reply:
Done :slightly_smiling_face: